test/unit/testCpu.js

```diff
@@ -55,18 +56,26 @@ exports.cpu_should_not_send_twice_to_same_series = function(finish) {
 };
 
 exports.cpu_should_calculate_percentage_usage = function(finish) {
```

Comment:
@grdryn I attempted using a generator for this section after you're comment, but I didn't think it made it easier to read the test.
How does this look instead using a sinon stub?